pkg/runtime/pubsub/errors.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/dapr/dapr/pkg/messages"
 )
 
-var ErrMessageDropped = errors.New("pubsub message dropped")
+var ErrMessageDropped = errors.New("pubsub message dropped") // TODO: remove this and use apierrors.PubSubMsgDropped
```

Comment:
todo?

Reply:
yes, this error was not created in my PR, so I just added it as an explicit todo since  ~13 errors were already being updated in this PR